src/pages/SignUp.jsx

```diff
@@ -8,59 +8,66 @@ import { Formik, Field, Form, ErrorMessage } from 'formik';
 import { Link, useNavigate } from "react-router-dom";
 import {signUpInputNames, signUpInputsInitialValues } from "../utils/constants.js";
 import { signUpSchema } from "../utils/formSchema.js";
-import { useAppProvider } from "../context/AppContext";
+import { useSelector, useDispatch } from "react-redux";
+import { sessionsActions } from "../store/sessions-slice";
+import { loginSignUpActions as signUpActions } from "../store/login-signUp-slice";
 
 const SignUp = () => {
-    const {
-        updateSessions,
-        showSignUpErrorMsg,
-        setShowSignUpErrorMsg,
-        passwordNotRepeatedError,
-        setPasswordNotRepeatedError,
-        saveLogged,
-    }  = useAppProvider();
+    const dispatch = useDispatch();
+    const signUpErrors = useSelector(state => state.loginSignUp.signUpErrors);
     const navigateTo = useNavigate();
     useEffect(() => {
-        setShowSignUpErrorMsg({ show: false, existingPart: ""});
-    }, [setShowSignUpErrorMsg]);
+        dispatch(signUpActions.setSignUpErrors({
+            passwordNotRepeated: false,
+            accountAlreadyExist: { show: false, existingPart: ""}
+        }));
+    }, [dispatch]);
```

Comment:
there's no need to do this, let's set those values as the default ones in the slice files and that's it